.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+If fixing bug, please document how to reproduce it.
```

Comment:
If fixing a bug, please document how to reproduce it. Tests should ideally use network namespaces to not interfere with the rest of the system.

If adding a feature, please describe why it's useful to add it.

Commits should be signed: Your Name [me@example.org](mailto:me@example.org), see https://www.kernel.org/doc/html/latest/process/submitting-patches.html#sign-your-work-the-developer-s-certificate-of-origin


Although the coding style for most tools is ancient, new code should follow the Linux kernel coding style. See https://www.kernel.org/doc/html/latest/process/coding-style.html.